pkg/metrics3/client.go

```diff
@@ -0,0 +1,464 @@
+package metrics3
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const (
+	ApiToken  string = "api"
```

Comment:
Just to have consistency with Anodot UI would name tokens:

*  Data Collection Token
* Access Key